packages/devtools_app/lib/src/timeline/flutter/timeline_service.dart

```diff
@@ -23,6 +22,10 @@ class TimelineService {
 
   final profilerService = CpuProfilerService();
 
+  Future<vm_service.Timestamp> vmTimelineMicros() async {
+    return await serviceManager.service.getVMTimelineMicros();
```

Comment:
follow up cl: rename 
serviceManager.service.getVMTimelineMicros to serviceManager.service.vmTimelineMicros

Reply:
this would require changing the vm_service API. We are just overriding the method in vm_service_wrapper

Reply:
oh right.